benchmarks/src/main/scala/benchmark/etl/ETLBenchmark.scala

```diff
@@ -88,14 +93,32 @@ class ETLBenchmark(conf: ETLBenchmarkConf) extends Benchmark(conf) {
   val sourceFormat = "parquet"
   val formatName = conf.formatName
   val writeMode = conf.writeMode
-  val tblProperties = if (formatName == "iceberg") {
-    s"""TBLPROPERTIES ('format-version'='2',
+  val tblProperties = formatName match {
+    case "iceberg" =>
+      s"""TBLPROPERTIES ('format-version'='2',
                        'write.delete.mode'='${writeMode}',
                        'write.update.mode'='${writeMode}',
                        'write.merge.mode'='${writeMode}')"""
-  } else {
-    ""
+    case "hudi" =>
+      // NOTE: This is only used to create single (denormalized) store_sales table;
+      //       as such we're reusing primary key we're generally using for store_sales in TPC-DS benchmarks
+      s"""TBLPROPERTIES (
+         |  type = '${if (writeMode == "copy-on-write") "cow" else "mor"}',
+         |  primaryKey = 'ss_item_sk,ss_ticket_number',
+         |  preCombineField = 'ss_sold_time_sk',
+         |  'hoodie.table.name' = 'store_sales_denorm_${formatName}',
+         |  'hoodie.table.partition.fields' = 'ss_sold_date_sk',
+         |  'hoodie.table.keygenerator.class' = 'org.apache.hudi.keygen.ComplexKeyGenerator',
+         |  'hoodie.parquet.compression.codec' = 'snappy',
+         |  'hoodie.datasource.write.hive_style_partitioning' = 'true',
+         |  'hoodie.sql.insert.mode'= 'non-strict',
```

Comment:
Following 3 options are actually shouldn't need to be specified explicitly, however in our recent round of testing we've discovered a regression ([HUDI-5499](https://issues.apache.org/jira/browse/HUDI-5499)) that prompted us to employ these here for a proper comparison